### ISSUE TITLE
Empty  `"outputs"` fields after `Hydrogen: export notebook`.

### ISSUE THREAD

**Issue by mRcSchwering** [nteract/hydrogen#1623]
**Description:**

Not sure if this is a bug, or whether I am using it wrong.
When I run "export notebook" the exported notebook never contains any output cells.
(Also markdown is not rendered, but I guess this is #1296).
Showing results in-line in Atom however works fine.

**Steps to Reproduce:**

1. Create a `.py` script with cells as described in [the docs](https://nteract.gitbooks.io/hydrogen/docs/Usage/NotebookFiles.html#cell-markers)
2. Add a cell which contains `print(1+1)`
3. Run `Hydrogen: Run All`
4. Run `Hydrogen: Export Notebook`
5. Check the created `.ipynb` file

- expected: `print(1+1)` cell has a output cell below which contains `2`
- actual: no output cell

**Versions:**

- OS: ubuntu 18.04
- Atom: 1.33.1
- Hydrogen 2.9.0

**Plugins:**

I only have Hydrogen installed and a few linter and auto-complete packages,
nothing else from Hydrogen.

**Logs:**

In the developer console, there are no logs showing up (Info, Warnings, Errors).
I checked _Enable Debug Messages_.

**Example:**

`script.py`

```
# %%
import matplotlib.pyplot as plt
import numpy as np

x = np.linspace(0, 20, 100)
plt.plot(x, np.sin(x))
plt.show()

# %%
print(1 + 1)

# %%
'''
# title

asd **asd**
'''

# markdown
# # title
# asd **asd**

# %%
print('Hello, world!')

# <codecell>
print('foo')

# In[1]
print('bar')
```

* `Hydrogen: Run All`
* `Hydrogen: Export Notebook`

`script.ipynb`

```
{
  "cells": [
    {
      "cell_type": "code",
      "source": [
        "import matplotlib.pyplot as plt\n",
        "import numpy as np\n",
        "\n",
        "x = np.linspace(0, 20, 100)\n",
        "plt.plot(x, np.sin(x))\n",
        "plt.show()\n"
      ],
      "outputs": [],
      "execution_count": null,
      "metadata": {
        "collapsed": false,
        "outputHidden": false,
        "inputHidden": false
      }
    },
    {
      "cell_type": "code",
      "source": [
        "print(1 + 1)\n"
      ],
      "outputs": [],
      "execution_count": null,
      "metadata": {
        "collapsed": false,
        "outputHidden": false,
        "inputHidden": false
      }
    },
    {
      "cell_type": "code",
      "source": [
        "'''\n",
        "# title\n",
        "\n",
        "asd **asd**\n",
        "'''\n",
        "\n",
        "# markdown\n",
        "# # title\n",
        "# asd **asd**\n"
      ],
      "outputs": [],
      "execution_count": null,
      "metadata": {
        "collapsed": false,
        "outputHidden": false,
        "inputHidden": false
      }
    },
    {
      "cell_type": "code",
      "source": [
        "print('Hello, world!')\n"
      ],
      "outputs": [],
      "execution_count": null,
      "metadata": {
        "collapsed": false,
        "outputHidden": false,
        "inputHidden": false
      }
    },
    {
      "cell_type": "code",
      "source": [
        "print('foo')\n"
      ],
      "outputs": [],
      "execution_count": null,
      "metadata": {
        "collapsed": false,
        "outputHidden": false,
        "inputHidden": false
      }
    },
    {
      "cell_type": "code",
      "source": [
        "print('bar')"
      ],
      "outputs": [],
      "execution_count": null,
      "metadata": {
        "collapsed": false,
        "outputHidden": false,
        "inputHidden": false
      }
    }
  ],
  "metadata": {
    "kernelspec": {
      "argv": [
        "python3",
        "-m",
        "ipykernel_launcher",
        "-f",
        "{connection_file}"
      ],
      "display_name": "Python 3",
      "language": "python",
      "name": "python3"
    }
  },
  "nbformat": 4,
  "nbformat_minor": 0
}
```

**Comment by wadethestealth** [comment]
@mRcSchwering This is not currently a feature of Hydrogen, but could prove useful to have. I am going to mention @BenRussert to see if it is something that should be added. Also #1296 is a separate issue and has already been solved.

**Comment by kylebarron** [comment]
@mRcSchwering
This is issue https://github.com/nteract/hydrogen/issues/1503. Ideally `export-notebook` would also include outputs but exporting just code was an easier first step.

I had trouble saving kernel responses and matching them to cells. Currently they're only really stored in display bubbles, and it's hard to get outputs out of those. 

I was leaning towards trying to run each cell one at a time, save the results and pair it with the input code. That means, however, that all code has to be re run, which could take a lot of computation time. 

In any case, I'm one month through a five month backpacking trip without access to a computer, so I won't be able to do anything more than comment in the near future. 
The issue is that kernel responses, I.e. outputs, aren't stored anywhere other than in the display bubbles. And it's hard to get them
I found that it's not the easiest to store kernel

**Comment by wadethestealth** [comment]
Closed in favor of #1503.
